NosTests/EventTests.swift

```diff
@@ -50,9 +50,9 @@ final class EventTests: XCTestCase {
             ]
           ],
           "created_at": 1675264762,
-          "pubkey": "d0a1ffb8761b974cec4a3be8cbcb2e96a7090dcf465ffeac839aa4ca20c9a59e",
-          "id": "931b425e55559541451ddb99bd228bd1e0190af6ed21603b6b98544b42ee3317",
-          "sig": "31c710803d3b77cb2c61697c8e2a980a53ec66e980990ca34cc24f9018bf85bfd2b0669c1404f364de776a9d9ed31a5d6d32f5662ac77f2dc6b89c7762132d63"
```

Comment:
@cjorgensen I updated the `sampleContactListJSONString`'s because it was failing to verify with the given id and sig. I'm assuming this was because you pulled some real event JSON and then edited a bit for the test. Just wanted to confirm that was the case because otherwise it indicates a bug in the verification logic.

Reply:
That's correct.